Sources/AltSwiftUI/Source/Views/Shapes/Shapes.swift

```diff
@@ -0,0 +1,169 @@
+//
+//  Shapes.swift
+//  AltSwiftUI
+//
+//  Created by Nodehi, Jabbar on 2020/09/10.
+//  Copyright © 2020 Rakuten Travel. All rights reserved.
+//
+
+import UIKit
+
+class AltShapeView: UIView {
+    public var caShapeLayer = CAShapeLayer()
+}
+
+public protocol shape: View, Renderable {
```

Comment:
Maybe renaming this into `ShapeProtocol` would be better? I don't think lowercase type names should be allowed :(

Reply:
I think just `Shape` would be good

Reply:
Fixed, changed it to "Shape"